test/features/events/response.test.ts

```diff
@@ -148,6 +149,7 @@ it('ClientRequest: emits the "response" event upon the original response', async
   expect(response.status).toBe(200)
   expect(response.statusText).toBe('OK')
   expect(response.headers.get('x-response-type')).toBe('original')
+  expect(Reflect.get(response.headers, kRawHeaders)).toContainEqual(['x-response-type', 'original'])
```

Comment:
The only thing I don't quite like about this change is this assertion. We shouldn't tap into internal symbols anywhere. 

Is there any other way to check that the fix is working?

Reply:
I don't think so. This is what I'm planning to do in `nock` as well to extract the `rawHeaders` from the `response` instance in the `response` listener